### ISSUE TITLE
Search bar

### ISSUE THREAD

**Issue by cor** [unionlabs/union#2541]
we should re-enable the search bar component we have before, with two simple features to start with:

when you paste in a tx hash, it allows you to go to `explorer/transfer/<tx_hash>`
when you paste in an address, it allowes you to go to `explorer/address/<addr>` (blocked by #2540).

## Out of scope for this PR

- searching by block
- searching by tx hash
